connector/src/main/java/com/datastax/oss/cdc/CassandraSourceConnectorConfig.java

```diff
@@ -716,6 +728,22 @@ public AuthenticatorConfig getAuthenticatorConfig() {
         return authConfig;
     }
 
+    public String getOutputFormat() {
+        return globalConfig.getString(OUTPUT_FORMAT);
+    }
+
+    public boolean isAvroOutputFormat() {
+        return globalConfig.getString(OUTPUT_FORMAT).contains("avro");
```

Comment:
in these cases I'd prefer to have private constant fields and do the equals/switch comparison. This ensure that if we add a new value, we'll need to handle it explicitly

Reply:
sure. let me make it explicit. probably with an enum + throwing exception for unknown format.